src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.UI;
+
+/** duke.Main class to run Focus. */
+public class Duke {
+    /** Storage created for user. */
+    private final Storage storage;
+    /** Task list created for user. */
+    private final TaskList taskList;
+    /** UI created to interact with user. */
+    private final UI ui;
+
+    /** Creates Focus to set up the things needed. */
+    public Duke() {
+        ui = new UI();
+        Storage.createFolder();
+        storage = new Storage();
+        if (storage.retrieveTextFile()) {
+            taskList = new TaskList(storage.loadData());
+        } else {
+            taskList = new TaskList();
+        }
+    }
+
+    /**
+     * Gets response from Focus based on user's input.
+     *
+     * @param input User's input.
+     * @return Focus' response.
+     */
+    public String getResponse(String input) {
+        boolean exit;
```

Comment:
Perhaps having a more easily understandable variable name to make it sound more like a boolean?

Reply:
Alright noted, I have changed it to `isExit`. Thank you!